src/recommendations/implementation.md

```diff
@@ -0,0 +1,27 @@
+---
+group: product-recommendations
+title: Implementation Steps
+ee_only: True
+---
+
+To help you implement product recommendations on your storefront, use the following workflow:
+
+1. **Deploy data collection to production**
+   -  **Why** - The more data you collect the more accurate your recommendations will be. As soon as you start collecting data, you can pull that data into your non-production environment and experiment with different recommendation types on different pages.
+   -  **How** - [Install the Product Recommendations module]({{ page.baseurl }}/recommendations/install-configure.html).
+   -  **Notes** - Deploying data collection will not change your storefront's appearance or your shopper's experience. Only creating and deploying recommendation units will introduce new sections on your storefront. Do not create recommendation units until you customize your template.
```

Comment:
```suggestion
   -  **Notes** - Deploying data collection will not change your storefront's appearance or your shopper's experience. Only creating and deploying recommendation units will alter the customer experience on your storefront. Do not create recommendation units until you customize your template (<-INSERT LINK).
```
ALso add a blurb that you should test on pre-production before deploying to prod